File: hyeyoung/week12/과일장수.java
import java.util.*;
class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        int len = score.length;
        Arrays.sort(score);
        for(int i = 1; i <= len / m; i++ ){
            answer  += score[len - m*i] * m ;
        }
        
        return answer;
    }
}

File: seoyeon/week12/과일 장수.py
def solution(k, m, score):
    tmp = []       # 각 상자의 최소 점수
    count = len(score) // m  # 몇 상자
    score.sort(reverse = True)  # 정렬
    i = 1
    while count :
        if i > len(score) :
            break
        else :
            idx = m * i
            tmp.append(score[idx-1])
        count -= 1
        i += 1
    return sum(tmp) * m